Automattic/wp-calypso
# ESModules + Tree Shaking (take two)

Reverts the revert https://github.com/Automattic/wp-calypso/pull/21800.

**summary**
when disabling `add-module-exports`, I tested the node server in both docker and local dev.  Neither of the test environments had secrets though so user-bootstrapping code was never hit on the server.  The `require` statements are run lazily and it wasn't discovered that a `.default` was missing until after merging.

Instead of just adding a `.default` I've gone ahead and changed it to an `import` (as well as enabled tree shaking for the file in question).  All the fixes are in commit: https://github.com/Automattic/wp-calypso/pull/21805/commits/ac27198515bb9724d474d2b0a8810786f51f9f2a

**to test**
1. make sure you have a full `secrets.json`. you can check the FG for instructions for where to find the `wpcom_calypso_rest_api_key`.
2. build docker locally
3. test local docker container
